File: tests/human/x/fs/save_jsonl_stdout.fs
open System.Text.Json

type Person = { name:string; age:int }

let people = [
    { name = "Alice"; age = 30 }
    { name = "Bob"; age = 25 }
]

for p in people do
    let json = JsonSerializer.Serialize(p)
    printfn "%s" json

File: tests/human/x/go/save_jsonl_stdout.go
//go:build ignore

package main

import (
	"encoding/json"
	"fmt"
)

type Person struct {
	Name string `json:"name"`
	Age  int    `json:"age"`
}

func main() {
	people := []Person{{"Alice", 30}, {"Bob", 25}}
	for _, p := range people {
		b, _ := json.Marshal(p)
		fmt.Println(string(b))
	}
}
